WordPressAuthenticator/Unified Auth/View Related/Reusable Views/GravatarEmailTableViewCell.swift

```diff
@@ -8,23 +8,34 @@ class GravatarEmailTableViewCell: UITableViewCell {
     ///
     @IBOutlet private weak var gravatarImageView: UIImageView?
     @IBOutlet private weak var emailLabel: UITextField?
+    @IBOutlet private var containerView: UIView!
+
+    @IBOutlet private var containerViewMargins: [NSLayoutConstraint]!
+    @IBOutlet private var gravatarImageViewSizeConstraints: [NSLayoutConstraint]!
 
     private let gridiconSize = CGSize(width: 48, height: 48)
+    private let girdiconSmallSize = CGSize(width: 32, height: 32)
 
     /// Public properties
     ///
     public static let reuseIdentifier = "GravatarEmailTableViewCell"
     public var onChangeSelectionHandler: ((_ sender: UITextField) -> Void)?
 
+    override func awakeFromNib() {
+        super.awakeFromNib()
+        containerView.layer.cornerRadius = 8
+        containerView.layer.borderColor = UIColor.systemGray3.cgColor
```

Comment:
just checking, do we need these setups when `hasBorders` is `false` as well?

Reply:
I think this can be set up once, and we'd only need to update the border width to show/hide the border. But I added an update in 3c70e8845b2945c7e342f6b53cfda335ed63d52a to configure the container view in one place to make it easier to read (performance shouldn't be a concern here since we have only one row of this type in the table view).